src/qt/locale/bitcoin_fr.ts

```diff
@@ -1407,226 +1407,230 @@
     </message>
     <message>
         <source>Helpful tip of the day:</source>
-        <translation type="unfinished"></translation>
+        <translation>Conseil utile du jour :</translation>
     </message>
     <message>
         <source>Tip</source>
-        <translation type="unfinished"></translation>
+        <translation>Conseil :</translation>
     </message>
     <message>
         <source>Never share your wallet.dat file/your private key with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Ne partagez jamais votre fichier wallet.dat/votre clé privée avec quiconque.</translation>
     </message>
     <message>
         <source>For more advanced settings use the console in &apos;Help&apos; -&gt; &apos;Debug Window&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Pour des paramètres plus avancés, utilisez la console dans &apos;Aide&apos; -&gt; &apos;Fenêtre de débogage&apos;</translation>
     </message>
     <message>
         <source>Encrypt your wallet with a strong passphrase for maximum security</source>
-        <translation type="unfinished"></translation>
+        <translation>Chiffrez votre portefeuille avec une passphrase forte pour une sécurité maximale.</translation>
     </message>
     <message>
         <source>Make sure to keep your wallet updated.</source>
-        <translation type="unfinished"></translation>
+        <translation>Veillez à mettre à jour votre portefeuille.</translation>
```

Comment:
"Veuillez mettre à jour votre portefeuille" - Typo

Reply:
No, it's not a typo, but it does need to be reworded.